tfe/config_windows.go

```diff
@@ -0,0 +1,46 @@
+// +build windows
+
+package tfe
+
+import (
+	"path/filepath"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	shell         = syscall.MustLoadDLL("Shell32.dll")
+	getFolderPath = shell.MustFindProc("SHGetFolderPathW")
+)
```

Comment:
I don't think so... This file is actually copied from the Terraform repo and if you look closely, it tries to find the users `Application Data` directory and not the `Home` directory.

And since the `Application Data` directory can be redirected to a location outside of the users `Home` directory (which is sometimes done in corporate environments), finding the `Home` directory would not be a guaranteed path to finding the `Application Data` directory.